frontend/packages/dev-console/src/utils/pipeline-utils.ts

```diff
@@ -17,12 +25,17 @@ import {
   PipelineTaskRef,
 } from './pipeline-augment';
 import { pipelineFilterReducer, pipelineRunStatus } from './pipeline-filter-reducer';
+import { PIPELINE_SERVICE_ACCOUNT, SecretAnnotationId } from '../components/pipelines/const';
 
 interface Resources {
   inputs?: Resource[];
   outputs?: Resource[];
 }
 
+type ServiceAccountKind = {
+  secrets: { [name: string]: string }[];
+} & K8sResourceCommon;
+
```

Comment:
```suggestion

```
This exists in `frontend/public/module/k8s/index.ts` - I know Vikram created this, but we cannot be creating duplicate identifiers just _because_.

Reply:
updated types.ts file to use the correct type for `secrets` propery in serviceAccountKind.